package.json

```diff
@@ -25,20 +25,41 @@
     "electron-util": "^0.11.0"
   },
   "devDependencies": {
+    "@sindresorhus/tsconfig": "^0.3.0",
+    "@types/electron-settings": "^3.1.1",
+    "@types/semver": "^5.5.0",
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
     "electron": "^4.0.4",
     "electron-builder": "^20.38.5",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-config-xo-typescript": "^0.9.0",
     "husky": "^0.14.3",
+    "lint-staged": "^8.1.5",
     "prettier": "^1.16.4",
-    "pretty-quick": "^1.10.0",
+    "typescript": "^3.3.4000",
     "xo": "^0.24.0"
   },
+  "lint-staged": {
+    "*.ts": [
+      "prettier --write",
+      "xo --fix",
+      "git add"
+    ]
+  },
   "prettier": {
+    "bracketSpacing": true,
     "useTabs": false,
     "semi": false,
-    "bracketSpacing": true,
     "singleQuote": true
   },
   "xo": {
+    "extends": [
+      "xo-typescript",
+      "prettier/@typescript-eslint"
```

Comment:
IIRC you don't need to add `prettier/@typescript-eslint` when `prettier: true` is set below. Also see here: https://github.com/sindresorhus/caprine/blob/master/package.json#L59-L73

Reply:
Without that, I get the following errors:

`Expected indentation of 3 tabs but found 6 spaces.                 @typescript-eslint/indent`

Reply:
Can you try to set [`"space": true`](https://github.com/xojs/xo#space) on `xo`?

Reply:
Just tried that now and it doesn't change anything

Reply:
Interesting. I'll look into xo and xo-typescript as it seems like a bug.